api/templates/citizenship-status.xml

```diff
@@ -61,6 +61,13 @@
       <BornOnBase>
         <Answer>{{branch $citizenship.BornOnMilitaryInstallation}}</Answer>
       </BornOnBase>
+      <MilitaryBase>
+        <Comment></Comment>
+        <Name>{{text $citizenship.MilitaryBase}}</Name>
+      </MilitaryBase>
+      {{end}}
+      {{if radio $citizenship.CitizenshipStatus | eq "ForeignBorn"}}
```

Comment:
This looks like it is redundant with the block starting on line 60? i.e., you could remove the end/and if...ForeignBorn.

Reply:
I think I fixed it. Let me know if it looks good.